language/move-vm/test-utils/src/storage.rs

```diff
@@ -111,6 +111,33 @@ impl<'a, 'b, S: ResourceResolver> ResourceResolver for DeltaStorage<'a, 'b, S> {
     }
 }
 
+#[cfg(feature = "table-extension")]
+impl<'a, 'b, S: TableResolver> TableResolver for DeltaStorage<'a, 'b, S> {
+    fn resolve_table_entry(
+        &self,
+        handle: &TableHandle,
+        key: &[u8],
+    ) -> std::result::Result<Option<Vec<u8>>, Error> {
+        // n.b. Doesn't support table delta for now
+        self.base.resolve_table_entry(handle, key)
```

Comment:
The motivation sounds reasonable to me, and I'd love to see TableResolver being implemented for all of these mock storages. With that being said, I wonder if it would be better to leave these as `unimplemented!`? (I could be wrong though as I don't know exactly how you are planning to use this in its incomplete state.)

Reply:
The `DeltaStorage` is used in the txn replay tool, with `self.delta` "patched" upon `self.base`. So this should work as expected, lacking the ability to patch table related change set.